app/controllers/challenges_controller.rb

```diff
@@ -19,6 +20,26 @@ def create
     end
   end
 
+  def for_evaluation
+    @evaluable_challenges = Challenge.evaluable(current_user)
+
+    if @evaluable_challenges.any?
+      render :for_evaluation, status: :ok
+    else
+      render json: { resource: 'challenge', errors: [challenge: 'not found'] }, status: :not_found
```

Comment:
I feel like `:not_found` might not be right here. Maybe I should just return the empty collection? Otherwise the UI is going to crap out instead of just saying "nothing to evaluate"